tests/kokkos-based/gtest_fixtures.hpp

```diff
@@ -326,4 +326,10 @@ using blas2_signed_float_fixture  = _blas2_signed_fixture<float>;
 using blas2_signed_double_fixture = _blas2_signed_fixture<double>;
 using blas2_signed_complex_double_fixture = _blas2_signed_fixture<std::complex<double>>;
 
+// Run all tests for types defined above
+#define FOR_ALL_BLAS2_TYPES(TEST_DEF) \
+  TEST_DEF(double) \
+  TEST_DEF(float) \
+  TEST_DEF(complex_double)
```

Comment:
Awww, no `complex_float`?

Reply:
That is my fault because I started all tests using `double`, `float` and `complex<double>`, so Miko was following my setup :) 
I created a new issue: https://github.com/kokkos/stdBLAS/issues/190 to add `complex<float>` later

Reply:
@MikolajZuzek my preference would be to remove this macro. It is just 3 lines of code :) it does not save you much typing. Also, I think it adds another layer that one would have to find when looking at the tests. 
I think this gtest_fixtures should only contain the fixtures. If you really want this macro, maybe we can put it in another file called `test_macros.hpp` or similar so that it is clear the distinction.